.github/workflows/repo-sync.yml

```diff
@@ -58,3 +58,29 @@ jobs:
         git remote add upstream https://github.com/oracle/graal
         git pull --quiet upstream release/graal-vm/20.1
         git push --quiet origin oracle-20.1
+
+  mandrel-20-1-bump-JDK:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout 20.1 branch
+      uses: actions/checkout@v2
+      with:
+        ref: 20.1
+        fetch-depth: 0
+    - name: Check and update OpenJDK version
+      run: |
+        git config --local user.email "fzakkak@redhat.com"
+        git config --local user.name "Foivos Zakkak"
+        curl https://api.github.com/repos/AdoptOpenJDK/openjdk11-upstream-binaries/releases | jq -r 'first | .assets[] | .browser_download_url' > dl_urls.txt
```

Comment:
Would this API URL help by any chance?
```
$ curl -v https://api.adoptopenjdk.net/v3/binary/latest/11/ea/linux/x64/jdk/hotspot/normal/openjdk 2>&1 | grep 'location: '
< location: https://github.com/AdoptOpenJDK/openjdk11-upstream-binaries/releases/download/jdk-11.0.8%2B7/OpenJDK11U-jdk_x64_linux_11.0.8_7_ea.tar.gz
```
There is also https://api.adoptopenjdk.net/v3/binary/latest/11/ea/linux/x64/staticlibs/hotspot/normal/openjdk

for the static libs artifact.

Reply:
It removes the `jq` magic so it's probably easier to maintain, thanks!